main.js

```diff
@@ -0,0 +1,112 @@
+
+function justDoIt (name){
+    return name + "justDoIt";
+}
+console.log(justDoIt+" "+("Rawan"));
+
+//part1 
+function bigOrSmallString (name){ 
+    if (name.length>10){ 
+        return name + "word is loooooong";
+        console.log("word is loooooong")
+
+    }else{
+        return name = "This word is short";
+        console.log("This word is loooooong!")
+    }
+}
+//part 2 
+function oddOrEvenString (string){
+    if (string % 2==o) {
+    return "This string length is TOTALLY even!";
+    console.log("This string length is TOTALLY even!")
+    }else{
+        "This string's length is odd"
+        console.log(This string's length is odd..")
+}
+
+}
+//part3 
+function medianOfArray (array)
+   var array = [1,2,3,4,5,6,7,8,9,10];
+
+}
+console.log()
+
+
+//part4 //$a=array
+function sumArray (array){
+    var array = [1,2,3,4,5,6,7,8,9,10];
+return sumArray(array);
+
+console.log(array (sumArray))
+
+//part5
+var vowels = {array: Vowels ['a','e','o','i','u'],numVowels [1,2,3,4,5,6,7,8,9,10]};
+function vowelCount(str){
+return Vowels
+}
+console.log(vowels(array))
+
+//part6
+
+ function initials (Name){
+     name : FirstName LastName;
+    var initials = item.FirstName.charAt(0).toUpperCase + item.LastName.chartAt(0).toUpperCase();
+return 
+ }
+
+ console.log(initials)
+
+ //part7 
+
+ var day = prompt ("what day is today");
+If Else (
+    "Monday"){
+     alert ("Energize");
+    }If Else (
+    "Tuesday"){
+     alert ("Just getting started!");
+    }If Else {
+    "Wednesday"}
+    alert("Hump Hump!");
+If Else {
+    "Thursday"}
+alert ("Almost there!");
+If Else {
+    "Friday"}
+alert("Weeeeeee!");
+Else {
+    "Huh, sorry didn't get that"
+}
+//part 8
+var userenter prompt ("enter your train type")
+function traintype(){
+ prompt (`The (L) Train, The (N) Train, The (S)ix Train`)
```

Comment:
you use prompt here but you didn't store it or return the computer won't remember or know any input from the user